rotkehlchen/tests/db/test_history_events.py

```diff
@@ -179,6 +180,33 @@ def test_read_write_events_from_db(database):
                 assert event == expected_event
 
 
+def test_read_write_customized_events_from_db(database):
+    db = DBHistoryEvents(database)
+    history_data = {
+        1: ('TEST1', TimestampMS(1000), 1, None),
+        2: ('TEST2', TimestampMS(2000), 2, {HISTORY_MAPPING_KEY_STATE: HISTORY_MAPPING_STATE_CUSTOMIZED}),  # noqa: E501
+        3: ('TEST3', TimestampMS(3000), 3, {HISTORY_MAPPING_KEY_STATE: HISTORY_MAPPING_STATE_CUSTOMIZED}),  # noqa: E501
+    }
+
+    add_history_events_to_db(db, history_data)
+
+    filter_query_args = [
+        (HistoryEventFilterQuery, {'customized_events_only': True}),
+        (HistoryEventFilterQuery, {'customized_events_only': True, 'from_ts': Timestamp(3)}),
+    ]
+
+    expected_identifiers = [
+        ['TEST2', 'TEST3'],
+        ['TEST3'],
+    ]
```

Comment:
This fits in one line

```suggestion
    expected_identifiers = [['TEST2', 'TEST3'], ['TEST3']]
```

Reply:
I kept this multi-line since it grew into 4 tests and otherwise it's hard to read these expected results against input data sets.